pyspecProcScripts/phasing.py

```diff
@@ -753,7 +753,7 @@ def hermitian_function_test(
 
 
 def determine_sign(
-    s, indirect, signal_freq_range, direct="t2", signal_pathway=None, fl=None
+    s, signal_freq_range, direct="t2", signal_pathway=None, fl=None
```

Comment:
Here, I just implemented several of the corrections, since I needed to edit for other purposes.